plugins/logging/logging.cpp

```diff
@@ -61,7 +61,14 @@ LOGGING_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb)
 
 LOGGING_API void LOGGING_CALL logDbgLog(unsigned srcLen, const char * src)
 {
-    DBGLOG("%.*s", srcLen, src);
+    StringBuffer log(srcLen, src);
+    StringArray loglines;
+    log.replace('\r', ' ');
```

Comment:
Slightly strange windows lines will have spaces appended to the end, but fairly unlikely to cause problems.

Reply:
It seems like we should have a function built into jstring to do this

Reply:
Yes, but I couldn't find one.